.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     steps:
       - checkout
       - run: ci/config_auth.sh
-      - run: nox -s unit-3.8 system-3.8 cover
+      - run: PYTHON=3.8 ci/run_pip.sh
```

Comment:
What is the rationale for avoiding `nox` under CI?

Reply:
Good question. I forget why I did this. I'll switch back to `nox` and see if anything breaks.

Reply:
It was because `run_pip.sh` calls `run_tests.sh`, which sets up application default credentials. I removed some fixtures from `conftest.py` that injected manual credentials from a service account key file.